range_queries/sparse_table.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+Implementation of Sparse Table
+
+Running Time Complexity
+Build : O(NlogN)
+Range Query : O(1)
+*/
+
+
+#include<vector>
+#include<iostream>
+#include<algorithm>
+
+void computeLogs(int n, std::vector<int> &logs) {   // Util function to store pre-computed logs
```

Comment:
why not?

Reply:
See https://github.com/TheAlgorithms/C-Plus-Plus/runs/956040542#step:11:51.

Reply:
ah, ok

Reply:
Weird that is reporting as an error.
Never mind, then, sorry... 😄

Reply:
> Weird that it's being reporting as an error.

It is not an ISO-C++ standard error but rather a Google specs conformity error. 

What @arpanmukherjee had was correct. Whenever an input variable needs to be modified, the reference needs to be passed. What you suggested, to remove the `&` is completely wrong. From what you suggested, a copy of the variable would have been passed and the program would not have executed correctly at all. Please do not advise wrong in your reviews. 
Since the variable is getting modified, a pointer should be used. This is also suggested in the same error message you shared the link above.

Reply:
Sorry for the confusion about this.
I'll be more careful next time. 🙂